gnocchi/rest/cross_metric.py

```diff
@@ -17,18 +17,27 @@
 """Timeseries cross-aggregation."""
 import collections
 
-
 import daiquiri
 import iso8601
+import numpy
 import pandas
 import six
 
+from gnocchi import carbonara
 from gnocchi import storage as gnocchi_storage
 
 
 LOG = daiquiri.getLogger(__name__)
 
 
+agg_map = {'mean': numpy.nanmean,
```

Comment:
AGG_MAP would be better to quickly see it's a kinda module-wide constant

Reply:
will do